components/base/sensorcontrolled/spin.go

```diff
@@ -51,6 +51,9 @@ func (sb *sensorBase) Spin(ctx context.Context, angleDeg, degsPerSec float64, ex
 			return err
 		}
 	}
+
+	// pause and resume the loop to reset the control blocks
```

Comment:
Should this comment also include a thing about kicking in the wrong direction? I don't know this code well enough to have an informed opinion.

Reply:
yeah this comment/change is also intended to fix the "kick". updating here as well